### DISCUSSION TITLE
We are getting 504 errors in the 6.0.0 release

### DISCUSSION THREAD

**Discussion by rajeevsathish** [Sunbird-Ed/Community#603] [Bugs] [answered]
Recently we are facing random 504 error on some static files from the server 
<img width="1626" alt="Screenshot 2023-08-07 at 11 16 16 AM" src="https://github.com/Sunbird-Ed/Community/assets/5653717/4cd8521a-57f5-4e01-9b6b-46c6b0bf9c2d">
Details are as follow
<img width="582" alt="Screenshot 2023-08-07 at 11 17 21 AM" src="https://github.com/Sunbird-Ed/Community/assets/5653717/66cad9a0-8b2b-4c00-bf4c-dc1366138422">
CC: @vinukumar-vs , @mahesh-maney

**Comment by mahesh-maney** [comment]
We did face this issue on our DEV and UAT environment.
A bit on the issue : Connection are assigned as and when the requests arrive. There are situations when requests are too little and these connections go into a stale mode (unusable mode) and eventually end up in 504 errors.

On Nginx configuration we changed http 1.1 to http 1.0 and restarted Nginx.
After this we have not encountered the 504 erros.

@rajeevsathish @vinukumar-vs

**Comment by Krishnaj20** [comment]
@rajeevsathish - can you publish what is agreed to be done to address this issue?

**Comment by sravani-64** [comment]
@rajeevsathish, we are also facing same issue on our dev env running on 4.10 version, can you pls let us know if you are able to resolve it by changing Nginx configuration?
CC @gandham-santhosh

**Comment by chitranshu-keshav** [comment]
@rajeevsathish Can you please get back on this
cc @ashokreddy1208

**Reply by chitranshu-keshav** [reply]
@rajeevsathish awaiting your response
cc @ashokreddy1208

**Comment by sravani-64** [comment]
@mahesh-maney , we have made changes in proxy default configmaps and restarted nginx service. Still we are facing 504 errors, the most reproducible scenario is, when the user stays idle for 5mins on the page. Can you pls suggest if there is any way to fix this issue?
cc @gandham-santhosh ,@coolbung, @Renuka-s, @vishalchincholi1, @Nithin-ns24

**Comment by ashokreddy1208** [comment, marked as answer]
@rajeevsathish @santhosh-tg we are not getting 504 errors in 6.0.0 release, we can close this discussion
